### PR TITLE
add 2021-04-16 lang-team-apr-update

### PR DESCRIPTION
**Pull request by nikomatsakis**

### CODE REVIEW

File: posts/inside-rust/2021-04-16-lang-team-apr-update.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The minutes contain [links the tracking issues for each project](https://github.com/rust-lang/lang-team/blob/master/design-meeting-minutes/2021-04-14-Planning-Meeting.md#updates-from-active-groups-and-projects) and those issues contain status reports. Here are some interesting updates:
+
+* [never type update](https://github.com/rust-lang/lang-team/issues/60#issuecomment-814509681):
+    * we have a general plan for hybrid fallback that we think will allow us to stabilize `!` at long last without breaking existing crates
+* [improved closure capture (RFC 2229)](https://github.com/rust-lang/lang-team/issues/50#issuecomment-814526085):
+    * implementation is working quite well and migration is nearly implemented also
+* [ffi-unwind](https://github.com/rust-lang/lang-team/issues/19#issuecomment-814581675):
+    * C-unwind implementation has landed
+    * there is some ongoing twork to
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2021-04-16-lang-team-apr-update.md:32]:
```suggestion
    * there is some ongoing work to fix some bugs in the initial implementation
    * looking at potentially moving to considering setjmp/longjmp
```

File: posts/inside-rust/2021-04-16-lang-team-apr-update.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    * implementation is working quite well and migration is nearly implemented also
+* [ffi-unwind](https://github.com/rust-lang/lang-team/issues/19#issuecomment-814581675):
+    * C-unwind implementation has landed
+    * there is some ongoing work to fix some bugs in the initial implementation
+    * looking at potentially moving to considering setjmp/longjmp
+* [nested pattern stabilization](https://github.com/rust-lang/rust/pull/83386#issuecomment-819719603) has almost completed
+
+## Upcoming design meetings
+
+We planned three design meetings for March. Our meetings are open for anyone to join and observe. They are also typically recorded and posted to YouTube. Ping nikomatsakis or joshtriplett for info about attending.
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2021-04-16-lang-team-apr-update.md:38]:
```suggestion
We planned two design meetings for April. Our meetings are open for anyone to join and observe. They are also typically recorded and posted to YouTube. Ping nikomatsakis or joshtriplett for info about attending.
```

File: posts/inside-rust/2021-04-16-lang-team-apr-update.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+* April 21 -- proposed "wasm" ABI ([lang-team#90](https://github.com/rust-lang/lang-team/issues/90)), featuring special guest Alex Crichton
+* April 28 -- generators ([lang-team#92](https://github.com/rust-lang/lang-team/issues/92)), featuring special guest Esteban Küber
+
+## Design meeting expectations
+
+* The document for the meeting must be prepared by the triage meeting on Tuesday and posted to the tracking issue.
+    * If it is not sent out by then, the meeting will be canceled. This gives folks 24 hour notice.
+* There is no expectation that people will read the document before the meeting. The meeting will begin with a recap of the document.
+    * However, there is no rule **against** reading the document beforehand and provided feedback or advice on how to improve it.
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2021-04-16-lang-team-apr-update.md:48]:
```suggestion
    * However, there is no rule **against** reading the document beforehand and providing feedback or advice on how to improve it.
```

**Comment by nikomatsakis** [comment]
Thanks for catching those, @Mark-Simulacrum. I updated the date to today.

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**